local-cluster/tests/local_cluster.rs

```diff
@@ -1083,10 +1084,15 @@ fn test_incremental_snapshot_download_with_crossing_full_snapshot_interval_at_st
         )
         .unwrap();
 
+    info!("Stop the test validator.");
+    let validator_info = cluster.exit_node(&validator_identity.pubkey());
+
+    info!("Purge the test validator bank snapshots.");
+    purge_old_bank_snapshots(validator_snapshot_test_config.bank_snapshots_dir, 0);
```

Comment:
Without purging the old bank snapshots, the test incremental_snapshot_download_with_crossing_full_snapshot_interval_at_startup fails because creating hardlink fails with file existing error.  

Sure, I will add the comment.

Reply:
added the comment